Tinylog/Views/Cells/TaskTableViewCell.swift

```diff
@@ -39,49 +44,43 @@ final class TaskTableViewCell: GenericTableViewCell {
                 fetchRequestCompleted.sortDescriptors = [positionDescriptor]
                 fetchRequestCompleted.predicate  = NSPredicate(
                     format: "archivedAt = nil AND completed = %@ AND list = %@",
-                    NSNumber(value: true as Bool), currentTask!.list!)
+                    NSNumber(value: true as Bool), list)
                 fetchRequestCompleted.fetchBatchSize = 20
                 let resultsCompleted: NSArray = try managedObjectContext.fetch(
                     fetchRequestCompleted) as NSArray
 
                 let total: Int = results.count - resultsCompleted.count
-                currentTask?.list!.total = total as NSNumber?
+                list.total = total as NSNumber?
 
-                checkBoxButton.circleView?.layer.borderColor = UIColor(
-                    rgba: currentTask!.list!.color!).cgColor
+                checkBoxButton.circleView?.layer.borderColor = UIColor(rgba: color).cgColor
                 checkBoxButton.checkMarkIcon?.image = checkBoxButton.checkMarkIcon?.image?.imageWithColor(
-                    UIColor(rgba: currentTask!.list!.color!))
+                    UIColor(rgba: color))
             } catch let error as NSError {
                 fatalError(error.localizedDescription)
             }
 
             updateFonts()
 
-            taskLabel.activeLinkAttributes = [
-                kCTForegroundColorAttributeName as AnyHashable: UIColor(
-                    rgba: currentTask!.list!.color!)]
+            taskLabel.activeLinkAttributes = [NSAttributedString.Key.foregroundColor: UIColor(rgba: color)]
 
-            if let boolValue = currentTask?.completed?.boolValue {
+            if let boolValue = task.completed?.boolValue {
                 if boolValue {
                     checkBoxButton.checkMarkIcon!.isHidden = false
                     checkBoxButton.alpha = 0.5
                     taskLabel.alpha = 0.5
-                    taskLabel.linkAttributes = [
-                        kCTForegroundColorAttributeName as AnyHashable: UIColor.lightGray]
+                    taskLabel.linkAttributes = [NSAttributedString.Key.foregroundColor: UIColor.lightGray]
                 } else {
                     checkBoxButton.checkMarkIcon!.isHidden = true
                     checkBoxButton.alpha = 1.0
                     taskLabel.alpha = 1.0
-                    taskLabel.linkAttributes = [
-                        kCTForegroundColorAttributeName as AnyHashable: UIColor(
-                            rgba: currentTask!.list!.color!)]
+                    taskLabel.linkAttributes = [NSAttributedString.Key.foregroundColor: UIColor(rgba: color)]
                 }
             }
+            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/7d74fdd396913e4e42ba0446eaceaffef14d4bd5/Tinylog/Views/Cells/TaskTableViewCell.swift#L79"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TaskTableViewCell.swift:79</code></td>
    </tr>
  </tbody>
</table>